python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -602,42 +600,17 @@ def get_2d_run_length_encoded_partitions(
 
 def get_freshness_info(
     asset_key: AssetKey,
-    freshness_policy: FreshnessPolicy,
     data_time_resolver: CachingDataTimeResolver,
-    asset_graph: AssetGraph,
 ) -> "GrapheneAssetFreshnessInfo":
     from ..schema.freshness_policy import GrapheneAssetFreshnessInfo
 
     current_time = datetime.datetime.now(tz=datetime.timezone.utc)
 
-    latest_record = data_time_resolver.instance_queryer.get_latest_materialization_record(asset_key)
-    if latest_record is None:
-        return GrapheneAssetFreshnessInfo(
-            currentMinutesLate=None,
-            latestMaterializationMinutesLate=None,
-        )
-    latest_materialization_time = datetime.datetime.fromtimestamp(
-        latest_record.event_log_entry.timestamp,
-        tz=datetime.timezone.utc,
-    )
-
-    used_data_times = data_time_resolver.get_used_data_times_for_record(
-        asset_graph=asset_graph, record=latest_record
-    )
-
-    current_minutes_late = freshness_policy.minutes_late(
-        evaluation_time=current_time,
-        used_data_times=used_data_times,
-    )
-
-    latest_materialization_minutes_late = freshness_policy.minutes_late(
-        evaluation_time=latest_materialization_time,
-        used_data_times=used_data_times,
-    )
-
     return GrapheneAssetFreshnessInfo(
-        currentMinutesLate=current_minutes_late,
-        latestMaterializationMinutesLate=latest_materialization_minutes_late,
+        currentMinutesLate=data_time_resolver.get_current_minutes_late(
+            asset_key, evaluation_time=current_time
+        ),
+        latestMaterializationMinutesLate=None,
```

Comment:
fyi: @bengotow -- we were not using this value anywhere so I set it to None instead of maintaining the code to populate it. I assume it'd be fine to just remove this field from the GrapheneAssetFreshnessInfo entirely? Just didn't want to make this PR larger than necessary